src/Microsoft.Diagnostics.Monitoring/Configuration/AggregateSourceConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Diagnostics.NETCore.Client;
+
+namespace Microsoft.Diagnostics.Monitoring.Configuration
```

Comment:
Remove "Configuration" from the namespace to be consistent with other configurations.